### PR TITLE
docs: Updated the Podman documentation

### PR DESCRIPTION
**Pull request by AxelSilverdew**
The podman documentation was pretty out of date, so I updated it and added a section about fixing SELinux errors that might crop up.

### CODE REVIEW

File: docs/tutorials/podman.rst [hunk 1]
```diff
@@ -17,36 +18,63 @@ Enabling Podman
...
 
+If you're running on a system that has ``SELinux``, you might run into a few errors that cause rootless containers to not be able to read configuration files.
 
-Enabling podman-compose
-~~~~~~~~~~~~~~~~~~~~~~~
+Instead of disabling SELinux, you could instead apply the appropriate labels to the tutor root directory. This can be done as follows::
 
-``podman-compose`` is available as a package from PyPI, and can thus be installed with ``pip``. See `its README <https://github.com/containers/podman-compose/blob/devel/README.md>`_ for installation instructions. Note that if you have installed Tutor in its virtualenv, you'll need to run ``pip install podman-compose`` in that same virtualenv.
+  # Set the SELinux type of the tutor root directory and all of it's subdirectories to `container_file_t`
+  $ sudo semanage fcontext -a -t container_file_t "/home/tutor/.local/share/tutor(/.*)?"
```
Comment (regisb) [inline docs/tutorials/podman.rst:59]: Replace "/home/tutor/.local/share/tutor" by "$(tutor config printroot)". And the note below can be removed.

File: docs/tutorials/podman.rst [hunk 2]
```diff
@@ -17,36 +18,63 @@ Enabling Podman
...
+.. note::
+   As with the previous ``alias`` command, if you'd like to make the ``DOCKER_HOST`` variable permanent, you should put the entire export command in your ``bashrc`` or equivalent file.
+
+Fixing SELinux Errors
+~~~~~~~~~~~~~~~~~~~~~
+
+.. warning::
+   Disabling ``SELinux`` or setting it to *permissive mode* on your system is **highly discouraged and will render your system vulnerable.**
 
+If you're running on a system that has ``SELinux``, you might run into a few errors that cause rootless containers to not be able to read configuration files.
```
[the remark this answers is not in the document]
Comment (AxelSilverdew) [reply docs/tutorials/podman.rst:54]: Actually, I should definitely reword that.

They will 100% run into errors. I'll try to grab some lines from my audit log and add them as reference.

File: docs/tutorials/podman.rst (hunk 1 above)
Comment (AxelSilverdew) [reply docs/tutorials/podman.rst:59]: Good point! I'll fix that real quick.

**Comment by AxelSilverdew** [comment]
@regisb Should be good now I think.

**Pull request merged**